charts/castai-cluster-controller/templates/secret.yaml

```diff
@@ -1,3 +1,5 @@
+{{- if or .Values.apiKey .Values.castai.apiKey }}
```

Comment:
Don't you need to nest it here as well?

`{{- if or (.Values.apiKey ) (.Values.castai.apiKey) }}`